specifyweb/frontend/js_src/lib/components/Syncer/xmlToJson.ts

```diff
@@ -50,7 +50,7 @@ export const xmlToJson = (element: Element): XmlNode => ({
 /**
  * Reverse conversion to JSON
  */
-export function jsonToXml(node: XmlNode): Element {
+export function jsonToXml(node: any): Element {
```

Comment:
please refrain from using `any` in our codebase.
`any` disabled type-checking, which negates the benefits of TypeScript

you can read a quick reference guide on TypeScript if you wish to be more familiar with TypeScript concepts: https://www.typescriptlang.org/docs/handbook/typescript-in-5-minutes.html
you can also use chatgpt to explain TypeScript errors

if you are using VS Code, you can also install this extension to make the error messages more readable: https://marketplace.visualstudio.com/items?itemName=yoavbls.pretty-ts-errors

Reply:
Aha, I didn't think my code would be reviewed yet 😅 I promise this was a temporary hack as I was still figuring out the xml types.

Reply:
Ah, that's partially my mistake, I didn't mean to trigger a code review. Nevertheless, these resources by Max are good starting points!